### PR TITLE
US15, US28, and their test files

### PR DESCRIPTION
**Pull request by mmccree1**

### CODE REVIEW

File: src/us15.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Created on 10/24/20
+@author:   Michael McCreesh
+Pledge:    I pledge my honor to abide by the Stevens Honor System
+SSW555 - Sprint 2
+'''
+
+'''Goal: Make sure there are no more than 15 siblings per family'''
+
+from Gedcom import Gedcom, Family
+from datetime import date, datetime
```
Comment (ming1in) [inline src/us15.py]: not using `datetime` import should delete

File: src/us28.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+'''Goal: List siblings in age order'''
+
+from Gedcom import Gedcom, Family
+from datetime import date, datetime
+
+def sibList(fam):
+    kidList = fam.child
+    
+    n = len(kidList) 
+    for i in range(n-1):
```
Comment (ming1in) [inline src/us28.py:17]: should just be able to do this `for child in fam.child`

File: src/us28.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+from datetime import date, datetime
+
+def sibList(fam):
+    kidList = fam.child
+    
+    n = len(kidList) 
+    for i in range(n-1): 
+        for j in range(0, n-i-1): 
+            day1 = datetime.strptime(kidList[j].birth, '%d %b %Y')
+            day2 = datetime.strptime(kidList[j+1].birth, '%d %b %Y')
```
Comment (ming1in) [inline src/us28.py]: just merged in changes to Gedcom so all dates should be parsed to datetime

**Review by ming1in — APPROVED** [review]
looks really good, just some small comment about some potential improvements. Before you merge, add your tests to the test_suite.py file and run it.

**Review by samanthainneo99 — APPROVED** [review]
looks good to me

**Pull request merged**